packages/tools/release-website.sh

```diff
@@ -71,7 +71,13 @@ cd "$JOPLIN_ROOT_DIR"
 CROWDIN_PERSONAL_TOKEN="$CROWDIN_PERSONAL_TOKEN" yarn crowdinDownload
 yarn buildWebsite
 
+
 cd "$JOPLIN_WEBSITE_ROOT_DIR"
+
+# yarn buildWebsite removes docs/plugins. Restoring the original version
+# significantly decreases the size of commits.
+git restore docs/plugins || true
```

Comment:
```suggestion
git restore docs/plugins
```

Reply:
What is the `|| true` for? I guess for `git commit` is that if there's no changes it returns an error (which we should ignore)?

Reply:
That was the original logic (or if the changes were already staged by `buildWebsite`). However, there should always be changes as `buildWebsite` seems to remove the `plugins` directory.